lib/helpers/utils.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
I'm mapping and joining because I want every word in the string to be capitalised, which I don't think any of those methods do. 

CSS only would be nice, but my main use for this function is to make the url match the database entries exactly.